docs/integrations/unity-catalog-spark.md

```diff
@@ -0,0 +1,179 @@
+# Unity Catalog and Spark
+
+This page explains how to create Unity Catalog tables and volumes with Apache Spark™.
+
+Spark is a great engine when your data is organized in Unity Catalog and vice versa.  
+
+* Neatly organizing data in tables and volumes in the Unity Catalog hierarchy makes it a lot easier to write Spark code.  
+* Make it easier to decouple business logic from file paths.  
+* Provides easy access to different file formats without end users needing to know how the data is stored.
+
+Let’s examine how to use Spark with Unity Catalog and discuss the advantages in more detail.
```

Comment:
We need a major section upfront explaining how to set up spark 3.5.3 and delta 3.2.1 with the right configurations.